### PR TITLE
Add Helm3 client and chart repos to Leo image

### PR DESCRIPTION
**Pull request by kyuksel**
[JIRA subtask](https://broadworkbench.atlassian.net/browse/IA-2136)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,5 +22,15 @@ RUN mkdir /leonardo
 COPY ./leonardo*.jar /leonardo
 COPY --from=helm-go-lib-builder /helm-go-lib-build/helm-scala-sdk/helm-go-lib /leonardo/helm-go-lib
 
+# Install the Helm3 CLI client from the source because installing it via the RHEL package managing didn't work
```
Comment (Qi77Qi) [inline Dockerfile]: I think installing from script is a bit different from installing from source :)

**Review by Qi77Qi — APPROVED** [review]
👍

File: Dockerfile (hunk 1 above)
Comment (kyuksel) [reply Dockerfile]: picky picky 😛 
j/k you're right - corrected

**Comment by rtitle** [comment]
👍

**Pull request merged**